baker/GrapherBaker.tsx

```diff
@@ -308,7 +309,6 @@ const bakeGrapherPageAndVariablesPngAndSVGIfChanged = async (
             imageMetadataDictionary
         )
     )
-    console.log(outPath)
```

Comment:
We output the slug in the progress bar already, this doesn't add anything useful on top of that.